DDD.Functions/EventbriteSync.cs

```diff
@@ -0,0 +1,95 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using DDD.Core.AzureStorage;
+using DDD.Functions.Config;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Table;
+using Newtonsoft.Json;
+
+namespace DDD.Functions
+{
+    public static class EventbriteSync
+    {
+        [FunctionName("EventbriteSync")]
+        public static async Task Run(
+            [TimerTrigger("%EventbriteSyncSchedule%")]
+            TimerInfo timer,
+            ILogger log,
+            [BindEventbriteSyncConfig]
+            EventbriteSyncConfig config
+        )
+        {
+            if (config.Now > config.StopSyncingEventbriteFromDate)
+            {
+                log.LogInformation("EventbriteSync sync date passed");
+                return;
+            }
+
+            var ids = new List<string>();
+            var http = new HttpClient();
+            http.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", config.EventbriteApiKey);
+            var response = await http.GetAsync($"https://www.eventbriteapi.com/v3/events/{config.EventId}/orders");
+            response.EnsureSuccessStatusCode();
+            var content = await response.Content.ReadAsAsync<PaginatedEventbriteOrderResponse>();
+            content.Orders.ToList().ForEach(o => ids.Add(o.Id));
+            while (content.Pagination.HasMoreItems)
+            {
+                response = await http.GetAsync($"https://www.eventbriteapi.com/v3/events/{config.EventId}/orders?continuation={content.Pagination.Continuation}");
+                response.EnsureSuccessStatusCode();
+                content = await response.Content.ReadAsAsync<PaginatedEventbriteOrderResponse>();
+                content.Orders.ToList().ForEach(o => ids.Add(o.Id));
+            }
```

Comment:
This could be cleaned up by extracting a method that takes the URL as an argument. Functionality looks solid though.